src/validators/contactinformation.js

```diff
@@ -35,7 +35,7 @@ export default class ContactInformationValidator {
 
     let successful = 0
     for (const item of this.phoneNumbers) {
```

Comment:
Should this be `const phoneNumber of this.phoneNumbers`, to match the loop in the `validEmails` method above?

Reply:
^ @floresj

Reply:
👍 👍  Will change. For some reason, I wasn't getting notifications for this repo when you all were tagging me. Sorry for the delay, I swear I wasn't ignoring you all!